plugin/src/main/groovy/com/novoda/staticanalysis/internal/lint/LintConfigurator.groovy

```diff
@@ -53,14 +54,21 @@ class LintConfigurator implements Configurator {
     }
 
     private CollectLintViolationsTask createCollectViolationsTask(Violations violations) {
-        def outputFolder = new File(project.projectDir, 'build/reports')
         project.tasks.create('collectLintViolations', CollectLintViolationsTask) { collectViolations ->
-            collectViolations.xmlReportFile = new File(outputFolder, 'lint-results.xml')
-            collectViolations.htmlReportFile = new File(outputFolder, 'lint-results.html')
+            collectViolations.xmlReportFile = xmlOutputFile()
+            collectViolations.htmlReportFile = new File(defaultOutputFolder(), 'lint-results.html')
             collectViolations.violations = violations
         }
     }
 
+    private File xmlOutputFile() {
```

Comment:
sorry to be a PITA, but this could be a getter `getXmlOutputFile()`, so you can use it simply at L58 as `xmlOutputFile` property